### ISSUE TITLE
Yellow stars don't have sufficient color contrast

### ISSUE THREAD

**Issue by caitlynmayers** [caitlynmayers/dukes#175]
### Location
https://dev-dukes.pantheonsite.io/product/real-mayonnaise/

### Description
We need to figure out how to change the stars for the product reviews so they can stay yellow but have a black border applied to them in order to meet color contrast requirements. Any ideas how to easily do this? Otherwise we'll have to change them to a dark color that's going to not look very good.

### Current Results
<img width="345" alt="screen shot 2017-08-04 at 11 41 19 am" src="https://user-images.githubusercontent.com/1574420/28975915-dd2f7e4a-7909-11e7-879b-0d81a64026c9.png">
<img width="928" alt="screen shot 2017-08-04 at 11 41 23 am" src="https://user-images.githubusercontent.com/1574420/28975918-de8c2e14-7909-11e7-87f3-3a34122d11a2.png">
